### DISCUSSION TITLE
I can't change my primary email address because GitHub won't send me a verification email for an additional email address

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127545] [New to GitHub]
I've added the wrong email address as my primary GitHub email address. I did receive a verification email from GitHub for that email address, and I responded to it and that email address was verified. It's a valid email address for me but its the wrong address because it does not correspond to the email address I use for my Azure account in Visual Studio, and so Visual Studio won't recognize me as a GitHub user for that account, and grant me access to Copilot (the whole point of signing up to GitHub in the first place).

I then added the correct email address - for my Visual Studio Azure account - as an additional email address, intending to change my incorrect primary email address to the new additional email address. After I'd added the additional email address I clicked the link to resend the verification email, so I could verify the additional email address. After doing this I did get another email from GitHub, but this email was just to inform me that someone had added an additional email address to my account. There was no verification link included in the email for me to verify the additional email that I had just added. I tried several times to resend the verification email, but it never arrived. Each time I just got another copy of an email which told me that someone had added a new email address to my account, but no verification link. Consequently, I was not able to verify the second (correct) email address, and so I could not use it to replace my incorrect primary email address. 

What can I do to fix that please?

**Comment by HusanDev7** [comment]
#1 respons

**Reply by 223047** [reply]
Canhductu121@gmail.com

**Comment by bakhtiyorov9** [comment]
hg

**Reply by 223047** [reply]
84398650782
